po/wesnoth/en_GB.po

```diff
@@ -11889,3 +11889,8 @@ msgstr "Whiteboard Options"
 
 #~ msgid "KB"
 #~ msgstr "KB"
+
+# po: This is the new chat text indicator
```

Comment:
Revert this entire change - if this is how new strings were added, you would have to update every single language.